src/main/java/frc/robot/subsystems/drivetrain/Chassis.kt

```diff
@@ -392,7 +424,7 @@ object Chassis :
         block: SwerveRequest.FieldCentricFacingAngle.() -> SwerveRequest.FieldCentricFacingAngle
     ): Command {
         return applyRequest {
-            FieldCentricFacingAngle.withTargetDirection(closestReef.rotation).block()
+            FieldCentricFacingAngle.withForwardPerspective(SwerveRequest.ForwardPerspectiveValue.OperatorPerspective).withTargetDirection(closestReef.rotation.rotateBy(Rotation2d.k180deg)).block()
```

Comment:
oh I see. Maybe it's worth making  second FieldCentricFacingAngle object that matches the op perspective?

Also, you shouldn't be rotating here. The robot faces away from the reef when scoring